### DISCUSSION TITLE
Get snapshot version of metals

### DISCUSSION THREAD

**Discussion by peter-jerry-ye** [scalameta/metals#4670] [Q&A] [answered]
Sorry, this may be a dumb question, but how to configure to get the snapshot version of metals?

Because in the [official document](https://scalameta.org/metals/docs/integrations/new-editor) it is written: `coursier bootstrap org.scalameta:metals_2.13:0.11.9+161-c4f0f913-SNAPSHOT -o metals -f`, which does not work. Is there a repository for snapshot version or does that mean we have to build it locally?

**Comment by tgodzik** [comment, marked as answer]
Actually it should be `coursier bootstrap -r sonatype:snapshots org.scalameta:metals_2.13:0.11.9+161-c4f0f913-SNAPSHOT -o metals -f
`, we need to update that document

**Reply by peter-jerry-ye** [reply]
It works. Thank you for your help.

**Reply by savulchik** [reply]
It seems like sonatype:snapshots repository is not working anymore.

I managed to install snapshot versions of metals & metals-mcp using the following commands:
```
$ cs bootstrap -r https://central.sonatype.com/repository/maven-snapshots org.scalameta:metals_2.13:1.6.7+122-51f35ed4-SNAPSHOT -o metals
$ cs bootstrap -r https://central.sonatype.com/repository/maven-snapshots org.scalameta:metals-mcp_2.13:1.6.7+122-51f35ed4-SNAPSHOT -o metals-mcp
```

I had to look up the latest SNAPSHOT version on the page https://scalameta.org/metals/docs/ because searching snapshot versions on maven central seems not working too :)

**Reply by tgodzik** [reply]
Yeah, I had to update the logic and crawl the new snapshot repositories manually. The logic is in https://github.com/scalameta/metals/blob/main/metals-docs/src/main/scala/docs/Snapshot.scala
